ADOPTERS.md

```diff
@@ -72,4 +72,4 @@
 | [QBE](https://www.qbe.com/)                                              | [Daniel Steel](https://github.com/danielsteelqbe), [Pete Jespers](https://github.com/petejespersqbe)                                                                                                         | Developer portal allowing our global teams to explore and create applications, documentation and cloud infrastructure easily and quickly 🚀                                                                                                                                                                                                          |
 | [LogMeIn](https://www.logmein.com)                                       | [Lorenzo Orsatti](https://github.com/lorsatti)                                                                                                                                                               | Improve onboarding experience of new developers. Discover faster and painlessly developer documentation, API definitions and team information. Provide useful dev metrics in a central place. Provide easy-to-use templates for new services.                                                                                                        |
-| [Telstra](https://www.telstra.com.au)                                       | [@kiranpatel11](https://github.com/kiranpatel11), [JasonC](https://github.com/JasonC17)                                                                                                                                                              | Primary usage: software catalog and templates<br/> Emerging usage : TechDocs, Explore Ecosystem, TechRadar, etc                                                                                                   |
+| [Telstra](https://www.telstra.com.au)                                    | [@kiranpatel11](https://github.com/kiranpatel11), [JasonC](https://github.com/JasonC17)                                                                                                                      | Primary usage: software catalog and templates<br/> Emerging usage : TechDocs, Explore Ecosystem, TechRadar, etc                                                                                                                                                                                                                                      |
```

Comment:
we probably want to omit this change although it appears to just be a spacing fix?

Reply:
Hm this file is changed when I commit by the automatic linting so it's not letting me revert it back and commit the old version locally - is there some way to turn off the auto-linting when I commit?